src/boundingboxes.jl

```diff
@@ -49,13 +60,12 @@ boundingbox(geoms::AbstractVector{<:Geometry}) = boundingbox(boundingbox.(geoms)
 boundingbox(boxes::AbstractVector{<:Box{Dim}}) where {Dim} =
   boundingbox([point for box in boxes for point in extrema(box)])
 
-function boundingbox(points::AbstractVector{Point{Dim,T}}) where {Dim,T}
-  xmin = MVector(ntuple(i -> typemax(T), Dim))
-  xmax = MVector(ntuple(i -> typemin(T), Dim))
+function boundingbox(points::AbstractVector{Point{Dim,T}}, xmin::Vec{Dim, T} = Vec(ntuple(i -> typemax(T), Dim)), xmax::Vec{Dim, T} = Vec(ntuple(i -> typemin(T), Dim))) where {Dim,T}
   for p in points
     x = coordinates(p)
-    @. xmin = min(x, xmin)
-    @. xmax = max(x, xmax)
+    xmin = min.(x, xmin)
+    xmax = max.(x, xmax)
   end
   Box(Point(xmin), Point(xmax))
 end
```

Comment:
No need to write this method

Reply:
this is just a change of the existing method with less allocations, not a new method
The only addition is, that it allows chaining boundingbox calls without needing a intermediate array for multiple boxes.

Reply:
I fully agree with the usage of `@.` to reduce allocations of the existing method, but the change of arguments to accept an extra xmin and xmax seems unnecessary at this point in time.

We can do another future round of review and improvements after we fix the internal representation of vertices, which is more pressing.